### DISCUSSION TITLE
ISR Problem: Polling( periodic revalidation) is not good solution : better solution is web hook trigger

### DISCUSSION THREAD

**Discussion by loveyandex** [vercel/next.js#32911] [Show and tell]
suppose we have a e-commerce website and product page.. admin generate or update some field of product like price some days ...
but nextjs has to  periodically fetch products for all pages created because number of user for seeing that product is something like  1k per min or more...this cause nextjs requests extra and redundant in order to find updates but there is no update per user request in revalidation ...

solution:
nextjs(nextjs sever) must not fetch to get updates for page.. it should have a  api route(webbooks) to be able call by CMS to make aware nextjs to update which pages. it is efficient solution to prevent huge useless request to update.....

if u can not understand this new model, i am here to listen and explain more

thank you  to let us have better web

**Comment by loveyandex** [comment]
@ijjk @timneutkens @sokra @kdy1 @styfle

**Comment by loveyandex** [comment]
you should implement generate as we demand not periodically
it is very clear but u have mistake in your design

**Comment by loveyandex** [comment]
why you are quiet

**Comment by leerob** [comment]
Please continue the conversation on the RFC: https://github.com/vercel/next.js/discussions/11552#discussioncomment-646963

Thank you!
